tests/main/null/task.yaml

```diff
@@ -0,0 +1,6 @@
+summary: Dummy test case which does nothing
```

Comment:
I can change this to `tests/main/help`. Having nothing is not an option as explained above.